security/identity-and-access-management/oracle-identity-governance/webservice-samples/extend-access-ws/files/WebContent/wsdl/ExtendAccessWSv1.wsdl

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wsdl:definitions targetNamespace="http://samplews.oracle.com" xmlns:apachesoap="http://xml.apache.org/xml-soap" xmlns:impl="http://samplews.oracle.com" xmlns:intf="http://samplews.oracle.com" xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:wsdlsoap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+<!--WSDL created by Apache Axis version: 1.4
+Built on Apr 22, 2006 (06:55:48 PDT)-->
+ <wsdl:types>
+  <schema elementFormDefault="qualified" targetNamespace="http://samplews.oracle.com" xmlns="http://www.w3.org/2001/XMLSchema">
+   <element name="extendAccess">
+    <complexType>
+     <sequence>
+      <element name="userId" type="xsd:string"/>
+      <element name="eaUUID" type="xsd:string"/>
+     </sequence>
+    </complexType>
+   </element>
+   <element name="extendAccessResponse">
+    <complexType>
+     <sequence>
+      <element name="extendAccessReturn" type="xsd:string"/>
+     </sequence>
+    </complexType>
+   </element>
+  </schema>
+ </wsdl:types>
+
```

Comment:
Remove empty lines.

Reply:
Done. Note that this is an auto-generated file, only provided as an example.